docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -28,6 +28,13 @@ snowflake.role=role
 snowflake.warehouse=warehouse
 ```
 
+### Procedures
```

Comment:
The place seems wrong. This is under Configuration section.

Reply:
Its like that everywhere. You think I should move it into the SQL section .. I kinda think yes .. will have a look what that would mean.

Reply:
I moved it now.